File: backend/ngts/src/main/java/com/three/ngts/Entity/Phone.java
package com.three.ngts.Entity;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "phone")
public class Phone {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "phone_id")
    private Long phoneId;

    @Column(name = "phone_number")
    private String phoneNumber;

    @ManyToOne
    @JoinColumn(name = "user_id")
    private User userId;
}

File: frontend/next-generation-ticketing-system/src/Entity/Phone.ts
class Phone {
  phoneId: number;
  phoneNumber: string;
  user: number;

  constructor(phoneNumber: string, user: number, phoneId: number) {
    this.phoneId = phoneId;
    this.phoneNumber = phoneNumber;
    this.user = user;
  }
}

export default Phone;
